db-scheduler-ui-frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && cross-env VITE_API_BASE_URL=https://jobjugglerapi-936dd3770db4.herokuapp.com/api vite build",
```

Comment:
På sikt tror jeg vi må finne en annen måte å få "injecta" API-url'en inn i frontenden, for det skal jo bli generisk eller kunne konfigureres av de som bruker rammeverket

Reply:
fjerne denne